src/test/mir-opt/nll/region-subtyping-basic.rs

```diff
@@ -7,7 +7,9 @@
 
 #![allow(warnings)]
 
-fn use_x(_: usize) -> bool { true }
+fn use_x(_: usize) -> bool {
```

Comment:
Maybe rustfmt ran after --bless?

Reply:
hmm no, the mir dump below has this change in it

Reply:
Ah I did misread the log

> [mir-opt] mir-opt/nll/region-subtyping-basic.rs' panicked at
> 'Actual MIR output differs from expected MIR output
> /checkout/src/test/mir-opt/nll/region-subtyping-basic/**32bit**/rustc.main.nll.0.mir'

it's indeed just 32 bit not having gotten updated